internal/generator/renderValues.go

```diff
@@ -13,12 +13,12 @@ import (
 // RenderValues renders the values of the given release and produces a yaml string, processing any go template
 // directives found in the values.
 func RenderValues(release *v1alpha1.Release) (string, error) {
-	valuesStr, err := yaml.Marshal(release.Spec.Values)
-	if err != nil {
-		return "", fmt.Errorf("marshalling release values: %w", err)
-	}
+	buf := &bytes.Buffer{}
+	encoder := yaml.NewEncoder(buf)
+	encoder.SetIndent(2)
+	err := encoder.Encode(release.Spec.Values)
```

Comment:
Yes indeed! Fixed

Reply:
@davidmdm just FYI, instead of something like:
```
  return "", fmt.Errorf("failed to encode spec: %w", err)
```
we settled on using error-wrapping messages that use affirmative/infinitive tense, such as:
```
  return "", fmt.Errorf("encoding spec: %w", err)
```
in order to avoid multi-level messages that look like this:
```
  error: failed to encode spec: error doing this: error doing that: failed this: failed that
```
which would rather read as:
```
  error: encoding spec: doing this: doing that: this: that
```